src/groups/mqb/mqba/mqba_authenticator.cpp

```diff
@@ -0,0 +1,199 @@
+// Copyright 2025 Bloomberg Finance L.P.
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// mqba_authenticator.h                                           -*-C++-*-
+#include <mqba_authenticator.h>
+
+#include <mqbscm_version.h>
+
+/// Implementation Notes
+///====================
+/// TODO
```

Comment:
`TODO`, but let's make sure to document this as it gets more complicated.  The authentication workflow is pretty simple, but let's do our best to describe how it goes.